client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -26,7 +26,7 @@ const shouldNext = action => {
 	}
 
 	// is a network response, don't reissue
-	if ( data.data || data.error || data.headers ) {
+	if ( data.data || data.error || data.headers || data.progress ) {
```

Comment:
the reason I originally didn't simplify was to keep the pattern of the function present. as in, we have a list of conditions which can accept, reject, or do nothing. this is just the last one.

simplifying is okay, but I feel like the only benefit is a few fewer bytes transferred because it obscures the flow (at least in my head). heck, this probably already gets optimized to the simpler version